cmp/form/JsonField.js

```diff
@@ -27,17 +27,27 @@ import './JsonField.css';
  *   A JSON Editor
  *
  * @prop rest, see general properties for HoistField
+ *      Accepts props documented below as well as any properties supported by the CodeMirror api,
+ *      with the exception of 'value', which is managed by HoistField.
+ *      Provided CodeMirror props will override any editor properties preconfigured here.
+ *      See: https://codemirror.net/doc/manual.html#api_configuration for details.
  *
  * @prop width, width of field, in pixels
  * @prop height, width of field, in pixels
- * @prop lineWrapping, Whether field should scroll or wrap for long lines. Defaults to false
+ * @prop disabled, Whether the field is readonly, Defaults to false
```

Comment:
This prop is technically documented in HoistField, but we are pulling it out and setting on the codemirror editor's readOnly prop so I thought it might be clearer to include here.